ios/Capacitor/Capacitor/CAPUNUserNotificationCenterDelegate.swift

```diff
@@ -133,7 +131,9 @@ public class CAPUNUserNotificationCenterDelegate : NSObject, UNUserNotificationC
       "title": content.title,
       "subtitle": content.subtitle,
       "body": content.body,
-      "badge": content.badge ?? 1
+      "badge": content.badge ?? 1,
+      "data": content.userInfo["custom_data"],
+      "aps": content.userInfo["aps"],
```

Comment:
As this is iOS specific and will be inside data if you do the previous change, this will be duplicated, so better remove it.